kube/connrst-server-deployment.yaml

```diff
@@ -0,0 +1,50 @@
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx
+  labels:
+    app: nginx
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx
+  template:
+    metadata:
+      labels:
+        app: nginx
+    spec:
+      initContainers:
+        - name: create-content
+          image: alpine
+          command: ['/bin/sh', '-c']
+          args: ['dd if=/dev/urandom of=/usr/share/nginx/small-file.txt bs=1M count=4;']
+          volumeMounts:
+            - name: workspace
+              mountPath: /usr/share/nginx
+      containers:
+      - name: nginx
+        image: nginx:1.15.1-alpine
+        imagePullPolicy: "IfNotPresent"
+        ports:
+        - name: http
+          containerPort: 80
+        resources: {}
+          # limits:
+          #   cpu: "1"
+          #   memory: 1Gi
+          # requests:
+          #   cpu: "1"
+          #   memory: 1Gi
+        volumeMounts:
+          - name: workspace
+            mountPath: /usr/share/nginx
```

Comment:
/usr/share/nginx is not really a directory where to put tempfile. As it is for test we couldn't care less, but make sure the google support don't tell: "ah it's because of that !"

Reply:
Ok I'll put it in `/data` to avoid this confusion